GitUpKit/Core/GCFunctions.h

```diff
@@ -39,6 +39,8 @@ NSString* GCFileSystemPathFromGitPath(const char* string);
 NSURL* GCURLFromGitURL(NSString* url);
 NSString* GCGitURLFromURL(NSURL* url);
 
+int CGExchangeDataFromTempPathToPath(const char* tempPath, const char* path);
```

Comment:
Should be called `GCExchangeFileData(const char* path1, const char* path2)`: this function is unrelated to temp files.